### ISSUE TITLE
dos.h needs comments

### ISSUE THREAD

**Issue by ainslec** [mattiasgustavsson/dos-like#23]
Hi, love the library but it's kind of difficult to work out what all the functions do in dos.h without experimentation sometimes (yes - even with the source). It'd be nice to have comments in the file, or maybe even a standalone document that shows all the functions and their parameters.

**Comment by mattiasgustavsson** [comment]
I agree, it needs documentation, as there is none at the moment. It is something I plan to do when I have time for it, but it might be a while
